plugins/modules/eos_lacp.py

```diff
@@ -37,8 +37,7 @@
 }
 
 DOCUMENTATION = """module: eos_lacp
-short_description: Manage Global Link Aggregation Control Protocol (LACP) on Arista
-  EOS devices.
+short_description: Lacp resource module.
```

Comment:
IMHO  LACP resource module (but you can have these changes  with script in different PR )